README.md

```diff
@@ -1,21 +1,128 @@
 # cairo-profiler
-Profiler for cairo programming language &amp; starknet
 
-## Setup for development
+Profiler for Cairo programming language &amp; Starknet.
 
-You need to install: rust, go, protobuf, pprof, graphviz 
+## Installation
 
-## Building protobufs
-Protocol buffers are built with [prost-build](https://github.com/tokio-rs/prost/tree/master/prost-build)
-
-## Run profiling
+To install the latest version of `cairo-profiler`, run:
 
+```shell
+curl -L https://raw.githubusercontent.com/software-mansion/cairo-profiler/main/scripts/install.sh | sh
 ```
-cargo run
+
+If you want to install a specific a version, run the following command with the requested version:
+
+```shell
+curl -L https://raw.githubusercontent.com/software-mansion/cairo-profiler/main/scripts/install.sh | sh -s -- v0.1.0
```

Comment:
We need one more `if` (for the Windows users 💀) because the installation script will not work on windows